src/Stripe.net/Services/FileLinks/StripeFileLinkUpdateOptions.cs

```diff
@@ -0,0 +1,37 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class StripeFileLinkUpdateOptions : StripeBaseOptions, ISupportMetadata
+    {
+        /// <summary>
+        /// A future timestamp after which the link will no longer be usable.
+        /// </summary>
+        public DateTime? ExpiresAt { get; set; }
```

Comment:
you likely should have a `StripeFileLinkSharedOptions` class since they have 2 parameters in common. Not a huge deal though since it does work that way

Reply:
Ack. I've updated the PR to use it.